Unable to add system includes

Unable to include `/usr/include/glib-2.0` as header search path, leading to

```zig
error: failed to translate /home/ultracode/repos/zig/libvips-with-translate-c/src/vips.h:
/usr/include/vips/vips.h:83:10: fatal error: 'glib.h' not found
#include <glib.h>
```
Relevant part of my `build.zig`, all the variants of `include_dirs` isn't working

```zig
    // Create a step to translate `header.h`.
    const translate_step = TranslateC.create(b, .{
        .root_source_file = b.path("src/vips.h"),
        .target = target,
        .optimize = optimize,
    });
    translate_step.include_dirs.append(.{ .path = .{ .cwd_relative = "/usr/include/glib-2.0/" } }) catch @panic("OOM");

    // Create a Zig module from the result.
    const header_module = translate_step.createModule();
    header_module.addAfterIncludePath(.{ .cwd_relative = "/usr/include/glib-2.0/" });

    const main_module = b.createModule(.{
        .root_source_file = b.path("main.zig"),
        .target = target,
        .optimize = optimize,
        .imports = &.{
            // Add the translated module as an import for the main module.
            .{
                .name = "vips",
                .module = header_module,
            },
        },
    });
    main_module.addSystemIncludePath(.{ .cwd_relative = "/usr/include/glib-2.0/" });


}
````

The above patch works, made a PR in https://github.com/ziglang/translate-c/pull/21

But now I get
```C
/usr/include/vips/create.h:166:2: error: TODO: implement 'sentinel' attribute for functions
        G_GNUC_NULL_TERMINATED;
  ^
/usr/include/glib-2.0/glib/gmacros.h:326:47: note: expanded from here
#define G_GNUC_NULL_TERMINATED __attribute__((__sentinel__))
```
I guess it's still not ready yet, will be on the lookout for >= 95% test passing

--- a/build.zig.zon
+++ b/build.zig.zon
@@ -26,6 +26,8 @@
         "build.zig",
         "build.zig.zon",
         "src",
+        "lib",
+        "test/cases.zig",
         "LICENSE",
         "README.md",
     },

--- a/build/TranslateC.zig
+++ b/build/TranslateC.zig
@@ -140,6 +140,47 @@ pub fn addCheckFile(translate_c: *TranslateC, expected_matches: []const []const 
     );
 }
 
+pub fn addAfterIncludePath(translate_c: *TranslateC, lazy_path: LazyPath) void {
+    const b = translate_c.step.owner;
+    translate_c.include_dirs.append(.{ .path_after = lazy_path.dupe(b) }) catch
+        @panic("OOM");
+    lazy_path.addStepDependencies(&translate_c.step);
+}
+
+pub fn addSystemIncludePath(translate_c: *TranslateC, lazy_path: LazyPath) void {
+    const b = translate_c.step.owner;
+    translate_c.include_dirs.append(.{ .path_system = lazy_path.dupe(b) }) catch
+        @panic("OOM");
+    lazy_path.addStepDependencies(&translate_c.step);
+}
+
+pub fn addIncludePath(translate_c: *TranslateC, lazy_path: LazyPath) void {
+    const b = translate_c.step.owner;
+    translate_c.include_dirs.append(.{ .path = lazy_path.dupe(b) }) catch
+        @panic("OOM");
+    lazy_path.addStepDependencies(&translate_c.step);
+}
+
+pub fn addConfigHeader(translate_c: *TranslateC, config_header: *Step.ConfigHeader) void {
+    translate_c.include_dirs.append(.{ .config_header_step = config_header }) catch
+        @panic("OOM");
+    translate_c.step.dependOn(&config_header.step);
+}
+
+pub fn addSystemFrameworkPath(translate_c: *TranslateC, directory_path: LazyPath) void {
+    const b = translate_c.step.owner;
+    translate_c.include_dirs.append(.{ .framework_path_system = directory_path.dupe(b) }) catch
+        @panic("OOM");
+    directory_path.addStepDependencies(&translate_c.step);
+}
+
+pub fn addFrameworkPath(translate_c: *TranslateC, directory_path: LazyPath) void {
+    const b = translate_c.step.owner;
+    translate_c.include_dirs.append(.{ .framework_path = directory_path.dupe(b) }) catch
+        @panic("OOM");
+    directory_path.addStepDependencies(&translate_c.step);
+}
+
 fn make(step: *Step, options: Step.MakeOptions) !void {
     _ = options;
     const b = step.owner;
@@ -187,6 +228,11 @@ fn make(step: *Step, options: Step.MakeOptions) !void {
             b.cache_root, sub_path_dirname, @errorName(err),
         });
     };
+
+    for (translate_c.include_dirs.items) |include_dir| {
+        try include_dir.appendZigProcessFlags(b, &argv_list, step);
+    }
+
     try argv_list.append("-o");
     try argv_list.append(out_path);
 

--- a/test/cases.zig
+++ b/test/cases.zig
@@ -22,7 +22,12 @@ pub fn addCaseTests(
         test_run_translated_step.dependOn(&exe.step);
     }
 
-    var dir = try b.build_root.handle.openDir("test/cases", .{ .iterate = true });
+    var dir = b.build_root.handle.openDir("test/cases", .{ .iterate = true }) catch |err| {
+        const fail_step = b.addFail(b.fmt("unable to open test/cases: {s}\n", .{@errorName(err)}));
+        test_translate_step.dependOn(&fail_step.step);
+        test_run_translated_step.dependOn(&fail_step.step);
+        return;
+    };
     defer dir.close();
 
     var it = try dir.walk(b.allocator);

